operator/Makefile

```diff
@@ -13,6 +11,14 @@ all: metacontroller/deploy controller/deploy
 .PHONY: clean
 clean: controller/undeploy metacontroller/undeploy
 
+prep-release:
+	rm -rf output
+	mkdir output
+	kustomize build ./controller > ./output/controller.yaml
```

Comment:
❓ For the release, are we expecting to deliver to one yaml file or multiple as shown here?

Reply:
The release is made up of 3 files that are the output of `kustomize build` on those 3 directories. This follows the same pattern that metacontroller uses so people can include things separately as needed.
https://github.com/OctoConsulting/keip/releases/tag/operator_v0.1.0